release-notes/README.md

```diff
@@ -14,12 +14,16 @@ You can find release notes for all releases, including out-of-support releases,
 [8.0.7]: 8.0/8.0.7/8.0.7.md
 [6.0.32]: 6.0/6.0.32/6.0.32.md
 
-## Release Information
+## Download .NET
 
 * [Download .NET](https://dotnet.microsoft.com/download/dotnet)
-* [Releases Index][releases-index.json] -- Index for all release channels in JSON format
-* [dotnet-install scripts](https://learn.microsoft.com/dotnet/core/tools/dotnet-install-script)
 * [Installation docs](https://learn.microsoft.com/dotnet/core/install/)
+* [dotnet-install scripts](https://learn.microsoft.com/dotnet/core/tools/dotnet-install-script)
+
+## Release notes in JSON format
```

Comment:
Does this need it's own section? Also, the formats file spends as much time on markdown as it does on json though I think I get wanting to call attention to the new json files.

Reply:
I think separating out download info from extra stuff is useful. I suspect 90%+ of people looking at this page are not interesting in advanced scenarios.